Fishare.Client/Client.cs

```diff
@@ -13,18 +13,25 @@ public class Client {
         byte[] ident = new byte[25];
         
         public void Connect() {
-            string addrp = File.ReadAllText("config.txt");
+            string addrp = File.ReadAllText(Path.Combine(System.IO.Directory.GetCurrentDirectory(), "config.txt"));
             string[] splitted = addrp.Split(":");
             IPHostEntry ipHostInfo = Dns.GetHostEntry(splitted[0]);  
             IPAddress ipAddress = ipHostInfo.AddressList[0];  
+            if (ipHostInfo.AddressList.Length > 1) {
+            ipAddress = ipHostInfo.AddressList[1];
+            }
```

Comment:
Please use tabulation (tab to move line). Also you can use else instead of strict line above